setup.cfg

```
@@ -47,7 +47,7 @@ server =
     # Basic dependencies
     fastapi ~= 0.63.0
     uvicorn[standard] ~= 0.13.4
-    elasticsearch >= 7.4.0,<8.0.0
+    opensearch-py
```

are all versions compatible, should we put at least an upper limit?

---

Just one version of opensearch client is published, but we can set a version number
```suggestion
    opensearch-py ~= 1.0.0
```